infrastructure/running-a-validator/operational-tasks/general-management.md

```diff
@@ -1,4 +1,354 @@
+        ``` bash 
+        sudo cp -r ./consoles /etc/prometheus
+        sudo cp -r ./console_libraries /etc/prometheus
+        sudo chown -R prometheus:prometheus /etc/prometheus/consoles
+        sudo chown -R prometheus:prometheus /etc/prometheus/console_libraries
+        ```
+    === "Clean up"
+        ```bash
+        cd .. && rm -rf prometheus*
+        ```
+
+3. **Create `prometheus.yml` for configuration** - run this command to define global settings, rule files, and scrape targets:
+    ```bash
+    sudo nano /etc/prometheus/prometheus.yml
+    ```
+    Prometheus is scraped every 5 seconds in this example configuration file, ensuring detailed internal metrics. Node metrics with customizable intervals are scraped from port `9615` by default.
+    ``` yaml title="prometheus-config.yml"
+    --8<-- 'code/infrastructure/running-a-validator/operational-tasks/general-management/prometheus-config.yml'
+    ```
+    
+4. **Validate configuration with promtool** - use the open source monitoring system to check your configuration
+    ```bash
+    promtool check config /etc/prometheus/prometheus.yml
+    ```
+5. **Assign ownership** - save the configuration file and change the ownership of the file to `prometheus` user
+    ```bash
+    sudo chown prometheus:prometheus /etc/prometheus/prometheus.yml
+    ```
+
+### Start Prometheus
+
+1. **Launch Prometheus** - use the following command to launch Prometheus with a given configuration, set the storage location for metric data, and enable web console templates and libraries: 
+
+    ```bash
+    sudo -u prometheus /usr/local/bin/prometheus --config.file /etc/prometheus/prometheus.yml --storage.tsdb.path /var/lib/prometheus/ --web.console.templates=/etc/prometheus/consoles --web.console.libraries=/etc/prometheus/console_libraries
+    ```
+
+    If you set the server up properly, you should see terminal output similar to the following:
+
+    -8<-- 'code/infrastructure/running-a-validator/operational-tasks/general-management/terminal-ouput-01.html'
+
+2. **Verify access** - verify you can access the Prometheus interface by visiting the following address:
+    ``` bash
+    http://SERVER_IP_ADDRESS:9090/graph
+    ```
+
+    If the interface appears to work as expected, exit the process using `Control + C`.
+
+3. **Create new systemd service file** - this will automatically start the server during the boot process
+    ```bash
+    sudo nano /etc/systemd/system/prometheus.service
+    ```
+    Add the following code to the service file:
+
+    ```bash title="prometheus.service"
+    --8<-- 'code/infrastructure/running-a-validator/operational-tasks/general-management/systemd-config.md'
+    ```
+    Once you save the file, execute the following command to reload `systemd` and enable the service so that it will load automatically during the operating system's startup:
+
+    ```bash
+    sudo systemctl daemon-reload && systemctl enable prometheus && systemctl start prometheus
```

Comment:
```suggestion
    sudo systemctl daemon-reload && sudo systemctl enable prometheus && sudo systemctl start prometheus
```

Reply:
You may not *need* the sudo command here but if we want to be consistent with how the commands appear in other places, we can add